python/src/scipp/_convolution.py

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2021 Scipp contributors (https://github.com/scipp)
+# @author Simon Heybrock
+from ._scipp import core as _cpp
+from . import Variable
+from ._cpp_wrapper_util import call_func as _call_cpp_func
+
+
+def convolve(x: Variable, kernel: Variable) -> Variable:
+    """Multidimensional convolution.
+
+    The array is convolved with the given kernel.
+    Currently this does not handle boundaries and the output is thus smaller
+    than the input.
```

Comment:
It would be nice if the size was not changed. I appreciate that masking etc will not help here. Maybe we can create the convoluted output to the right size and work on a slice or something similar?

Reply:
You mean just zero init the bounds, until we have proper handling?